### PR TITLE
SFDCENG-925 : Community VF Configuration and Setup

### PR DESCRIPTION
**Pull request by rrajoria**

### CODE REVIEW

File: salesforce/latest/using/community.md [hunk 1]
```diff
@@ -136,4 +136,39 @@ You will see a **Salesforce Community Member Access** message. The result of the
 
       ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
 
-  Community Members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.
\ No newline at end of file
+  Community Members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.
```
Comment (jottley) [inline salesforce/latest/using/community.md:139]: This is not limited to cases.  Any Salesforce sObject or custom object that is used in a community can be used with the connector.

File: salesforce/latest/using/community.md [hunk 2]
```diff
@@ -136,4 +136,39 @@ You will see a **Salesforce Community Member Access** message. The result of the
...
+  
+## Community Visual Force Page Configuration and Setup
+
+To expose the connector in a Salesforce Community (Digital Workspace)we need to add a visual force page for the connector.
```
Comment (jottley) [inline salesforce/latest/using/community.md:143]: There is a space need after (Digital Workspace)

File: salesforce/latest/using/community.md [hunk 3]
```diff
@@ -136,4 +136,39 @@ You will see a **Salesforce Community Member Access** message. The result of the
...
       ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
 
-  Community Members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.
\ No newline at end of file
+  Community Members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.
+  
+## Community Visual Force Page Configuration and Setup
+
+To expose the connector in a Salesforce Community (Digital Workspace)we need to add a visual force page for the connector. 
+Visual Force Page must be added to your Salesforce organization using the following pattern:
```
Comment (jottley) [inline salesforce/latest/using/community.md:144]: There are no spaces in Visualforce

Comment (jottley) [reply salesforce/latest/using/community.md:144]: and to keep concepts consistent it should be called a template and not a pattern

File: salesforce/latest/using/community.md [hunk 4]
```diff
@@ -136,4 +136,39 @@ You will see a **Salesforce Community Member Access** message. The result of the
...
+
+The `id` attribute is required and can be set to any non-empty, non-null value.
+
+The `applicationName` attribute must be set to the API Name of the connected app created for the connector during the initial setup.
+
+The `width`, `height`, and `scrolling` attributes can be set to values that meet the use case.
+
+The most important attribute is the `parameters` attribute.  The value must be set to `{'community':'simple'}`.Without this value, the connector will surface the standard connector view. 
+
+The Visual force page will need to have the appropriate profiles added for it to be visible to the correct community users.
```
Comment (jottley) [inline salesforce/latest/using/community.md:164]: There are no spaces in Visualforce

File: salesforce/latest/using/community.md [hunk 5]
```diff
@@ -136,4 +136,39 @@ You will see a **Salesforce Community Member Access** message. The result of the
...
+
+**Example 1:**
+
+```xml
+<apex:page docType="html-5.0" standardController="Case">
+  <apex:canvasApp id="AlfCanvas3" applicationName="ACS_Salesforce" width="100%" height="450px" scrolling="auto" parameters="{'community':'simple'}"/>
+</apex:page>
+```
+
+Add the Visual Force Page to Specific record in Salesforce Communities.
```
Comment (jottley) [inline salesforce/latest/using/community.md:174]: There are no spaces in Visualforce

Comment (jottley) [reply salesforce/latest/using/community.md:174]: Reword to something like:

> Add the new Visualforce Page to the desired record detail in your Salesforce Community.

File: salesforce/latest/using/community.md (hunk 1 above)
Comment (rrajoria) [reply salesforce/latest/using/community.md:139]: This  line  is already present for previous section. Its not added by me.

File: salesforce/latest/using/community.md (hunk 2 above)
Comment (rrajoria) [reply salesforce/latest/using/community.md]: Added space

File: salesforce/latest/using/community.md (hunk 4 above)
Comment (rrajoria) [reply salesforce/latest/using/community.md]: Renamed to  Visualforce

File: salesforce/latest/using/community.md (hunk 5 above)
Comment (rrajoria) [reply salesforce/latest/using/community.md]: Updated

File: salesforce/latest/using/community.md (hunk 3 above)
Comment (rrajoria) [reply salesforce/latest/using/community.md]: updated

**Review by jottley — APPROVED** [review]

**Review by gravitonian — APPROVED** [review]
Built and looked at the additional content. Looks ok.

**Pull request merged**